### ISSUE TITLE
[Sharepoint] Complete tpwSafety Build 

### ISSUE THREAD

**Issue by KaroEngstrom** [cityofaustin/atd-data-tech#18805]
## Purpose
From #13651, we need to finalize the TPW Safety SharePoint build. Katherine H. has also indicated that her team wants to move items from their OneDrive and G: drive spaces into SharePoint. 
- Meeting set up for Tue 8/27/24 with Katherine H. and Chris W. from Safety
---
### New Site Overview
- Site: [tpwSafety](https://cityofaustin.sharepoint.com/sites/tpwSafety) - not yet linked from the TPW Hub
- No site pages yet beyond the landing page 
- Libraries moved over from the PWD site: 
![image](https://github.com/user-attachments/assets/4f4f46ea-5027-4818-94ef-b6e6b4baaab8)
- [Miro board](https://miro.com/app/board/uXjVKl4rjeo=/) for this site

### Requirements 
- from Katherine via Teams: 
     - create a ShareDrive to store all the Safety Office training, forms, and all materials that staff has stored on their cloud and G: drive
     - main goal: create a SharePoint space for storing training and PowerPoint materials in one location

**Comment by KaroEngstrom** [comment]
## Tue 8/27/24 Meeting 
- look at current site and content (Miro) 
- confirm purpose and timeline for building out this site
- scope needs in Miro and create libraries/folders/pages as needed 
- make plan for moving items over

**Comment by KaroEngstrom** [comment]
## Thu 9/5/24 
- huddled with Diana and Katherine to discuss how to update the Outlook calendar - Katherine now has the template and ability to add currently scheduled trainings 
- Chris called yesterday and Karo brought him up to speed on how the calendar works 
- site was streamlined down the basic info, team contacts, and the calendar
- Katherine will gather more pieces from her team in terms of sharing out documents to the site visitors down the line

**Comment by KaroEngstrom** [comment]
Closing this as we are waiting on the stakeholder to provide content for the site update and there is no established timeline. Will create a new issue once the site update and file storage issue come up from the stakeholder again.
